canonical-web-and-design/canonical.com
# Remove unused send cv template

## Done

- Removed unused template
- Removed only reference of string referenced in #98, making it redundant

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers (check the whole section)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)


## Issue / Card

Fixes #98
